src/v/kafka/server/group_manager.cc

```diff
@@ -219,7 +219,8 @@ ss::future<> group_manager::handle_partition_leader_change(
     p->loading = true;
     if (leader_id != _self.id()) {
         // TODO: we are not yet handling group / partition deletion
-        klog.trace("ignorning group/partition deletion on node {}", _self.id());
+        p->loading = false;
+        return ss::make_ready_future<>();
     }
```

Comment:
Please squash the first two commits

```suggestion
    if (leader_id != _self.id()) {
        // TODO: we are not yet handling group / partition deletion
        return ss::make_ready_future<>();
    }
    p->loading = true;
```

Reply:
Believe it or not, I've already tried placing the `loading` boolean after the initial if statement, however that doesn't solve the problem because [every partition](https://github.com/vectorizedio/redpanda/blob/282a3c52a1331d4a7e7e895420fb99bdd6a1648d/src/v/kafka/server/group_manager.h#L196) is created with `loading = true` and then that boolean is never flipped for the followers. Perhaps this means the problem is even deeper?

Reply:
Oh, how subtle. Then perhaps the least surprising expression of the semantic is:
```cpp
    p->loading = leader_id == _self.id();
    if (!p->loading) {
        // TODO: we are not yet handling group / partition deletion
        return ss::make_ready_future<>();
    }
```
But it begs the question as to whether initialisation of `p->loading` can be done directly at construction.